cmd/podman/shared/funcs.go

```diff
@@ -21,7 +22,10 @@ func GetAuthFile(authfile string) string {
 	if runtimeDir != "" {
 		return filepath.Join(runtimeDir, "containers/auth.json")
 	}
-	return ""
+	if !rootless.IsRootless() {
+		return fmt.Sprintf("/run/containers/%d/auth.json", rootless.GetRootlessUID())
+	}
+	return fmt.Sprintf("/run/user/%d/containers/auth.json", rootless.GetRootlessUID())
```

Comment:
`pkg/util.GetRootlessRuntimeDir` goes through al the trouble to set up a non-trivial value, and this just hard-codes one path. The two should work together somehow.

Reply:
Yes this is just a hack to get us to release, while the keyring path is fixed in containers/image.

Reply:
(For the record: `pkg/util.GetRootlessRuntimeDir` is part of libpod, and hard-codes different behavior than c/image — it tries much harder to find/create directories when the user has no access to `/run/user` and/or `/tmp`.)